Assets/Python/GreatPeople.py

```diff
@@ -444,27 +454,49 @@ def setup():
 		"Aischylos", # 5th BC
 		"Pheidias", # 5th BC
 		"Anyte Tegeatis", # 3rd BC
+		iIndustrial,
+		"Dionysios Solomos", # 19th
+		"Georgios Jakobides", # 19th
+		iGlobal,
+		"Nikos Kazantzakis", # 20th
+		"fMaria Callas", # 20th
+		"Odysseas Elytis", # 20th
+		"Iannis Xenakis", # 20th
+		"Mikis Theodorakis", # 20th
 	],
 	iGreatScientist : [
 		"Pythagoras", # 6th BC
 		"Sokrates", # 5th BC
 		"Demokritos", # 5th BC
 		"Anaxagoras", # 5th BC
 		"Hippokrates", # 5th BC
-		iClassical,
```

Comment:
Why did you remove this?

Reply:
I think all the names above the iClassical line are Classical Era people. In all other Greek GP categories, people from the 6th and 5th century BC are considered classical era.

Reply:
Oh okay, yeah that makes sense.